transformers/tests/utils.py

```diff
@@ -0,0 +1,40 @@
+import os
+import unittest
+
+from distutils.util import strtobool
+
+
+_should_run_slow_tests = None
+
+def should_run_slow_tests():
+    # Cache return value in a global variable
+    global _should_run_slow_tests
```

Comment:
given that the code below is so fast maybe we don't need to cache it?

(I don't know whether `global` variables are as much of a code smell in Python as they are in other languages)

Reply:
I rewrote it since then, I'll update the PR soon.